blazar_dashboard/content/leases/templates/leases/_widget_capabilities.html

```diff
@@ -146,15 +179,32 @@
         values.forEach(function(cv) {
             element.appendChild(new Option(cv, cv));
         });
-        setDefaultNodeType();
+        setResourcePropertyValues(element.id.slice(-1));
     }
 
     function setDefaultNodeType(){
         var nodeOptions = $("#resource-node-type");
-        console.log("getting node options in widget capabilitiesjs")
         try {
           nodeOptions.find('[value="compute_haswell"]').attr("selected","selected");
-          console.log("node options in widget capabilitiesjs found: " + nodeOptions);
+        } catch(err) {
+          console.error(err);
+        }
+    }
+
+    function setPropertyType(index, propType){
+        var nodeOptions = $('#criteria-resource_properties-value-' + index);
+        try {
+          nodeOptions.find('[value="' + propType + '"]').attr("selected","selected");
+        } catch(err) {
+          console.error(err);
+        }
+    }
+
+    function setConditionalType(index, nodeCondition){
+        var nodeConditions = $('#criteria-resource_properties-equality-' + index);
+        var conditions = {"==":"eq","<":"lt","<=":"le",">":"gt",">=":"ge","!=":"ne"}
```

Comment:
These apply to options in a resource property type like `node_type` = `compute_haswell` vs `hp_atom` instead of between different sets of resource properties like the previous example. I grabbed the set from the widget code that generates the list of resource properties. 
https://github.com/ChameleonCloud/blazar-dashboard/blob/chameleoncloud/rocky/blazar_dashboard/content/leases/widgets.py#L10